java/simple-performance-test/pom.xml

```diff
@@ -97,15 +95,6 @@
     </dependency>
   </dependencies>
 
-  <repositories>
```

Comment:
This is a duplicate definition. The sample does not compile with this here.

The snapshots repository is defined above. https://github.com/GoogleCloudPlatform/cloud-bigtable-examples/pull/154/files#diff-6de1b0be654f8f9c45ef8657c367c531R35